Assets/MixedRealityToolkit-Examples/Demos/UX/Lines/Scenes/LineGallery.unity

```diff
@@ -1213,6 +1213,7 @@ Transform:
   m_Children:
   - {fileID: 1420623844}
   - {fileID: 1491536846}
+  - {fileID: 1237288067}
```

Comment:
I take it that this removes the TMPro Description panel I put in?

Reply:
It does not! It updates the MRTK logo with the updated version I added (which is transparent, so I also removed the transparent logo you added that was identical to this new one).

Reply:
What did you think about the text resolution with the SDF?